### DISCUSSION TITLE
Cooperative mobile drag mode?

### DISCUSSION THREAD

**Discussion by cmahnke** [openseadragon/openseadragon#2596] [Q&A]
Google Maps has a so called [cooperative mode](https://developers.google.com/maps/documentation/javascript/interaction#gesturehandling_cooperative) which is really helpful on mobile devices: It disables the one touch drag in favour of keeping the (drag) navigation of the surrounding page working.
If you try to use one finger you get a message, that you need to use two fingers. The link also provides an example.
Has Openseadragon a similar feature? Or has anyone a code snippet to enable this?

**Comment by iangilman** [comment]
We don't have a feature like this but it is something people talk about from time to time. As far as I know we don't have an explicit enhancement issue filed for it... Feel free to add one!

Here's some discussion: 

https://github.com/openseadragon/openseadragon/issues/2151

https://github.com/openseadragon/openseadragon/issues/1791

I wasn't aware that Google Maps has a flag for this and a name for the feature. Following their lead might be a good way to cut through all the possibilities.

**Reply by cmahnke** [reply]
Thanks, I've added #2597 and provided some more examples, all map related, although OpenLayers can also be used to display IIIF pyramids.

**Reply by iangilman** [reply]
Great, thank you!
